File: DayThree/num_of_step.py
class Solution:
    def numberOfSteps(self, num: int) -> int:
        step = 0
        while num > 0:
            step += 1
            if num % 2 == 0:
                num //= 2
            else:
                num -= 1
        return step

s = Solution()
print(s.numberOfSteps(14))
print(s.numberOfSteps(8))

File: DayThree/num_of_step.swift
class Solution {
    func numberOfSteps(_ num: Int) -> Int {
        var step: Int = 0
        var numVar: Int = num
        while numVar > 0 {
            step += 1
            if numVar % 2 == 0 {
                numVar /= 2
            } else {
                numVar -= 1
            }
        }
        return step
    }
}